### DISCUSSION TITLE
the bookmark at static line? How bookmark can sticky accordingly code?

### DISCUSSION THREAD

**Discussion by HypeNeutron** [alefragnani/vscode-bookmarks#399] [Q&A] [answered]
That is Problem on me when refactor code.

**Comment by alefragnani** [comment, marked as answer]
Hi @RedNeutorn ,

This is probably the same issue with Formatters, like Prettier. The extension only detects (for now) user changes. I mean, when you add or delete lines. The extension needs to be updated to properly detect and update the bookmark positions.

When originally reported #118, I’ve created a workaround, but it only _turned off_ the sticky behavior. But the idea of the workaround was to avoid moving the bookmarks to wrong positions.

This is on the roadmap.

Hope this helps
